documentation/SolutionsRegistryFormat.md

```diff
@@ -59,13 +65,24 @@ The `settingsHandlers` block is unique and one of the most important blocks in t
         "options": {
             "filename": "${{environment}.HOME}\\mySettings.ini"
         },
+        "supportedSettings": [
+            "appsettingA",
+            "appsettingB",
+            ...
+            "appsettingZ"
+        ],
         "capabilities": []
     }
 }
 ```
 
 The important thing to notice here is that this solution example has two references to settingsHandler - one XMLHandler which has been given a reference `myconf` and an INIHandler referred to as `otherconf`.
 
+Note also the `supportedSettings` option. If a solution only has a single settingsHandler block, all the settings will be passed to that handler by default. But in case there are multiple settingsHandlers, the system needs some way of determining which settings to apply to which handler. The `supportedSettings` directive is used for this:
+* If a `supportedSettings` option is supplied, only those settings listed there will be applied to the settingsHandler
+* If one or more settingsHandlers do not contain a `supportedSettings` directive, all settings will be applied to those handler
```

Comment:
"those handler" ungrammatical. Also, this case is dangerous - we should instead insist that if there are multiple settingsHandlers, they must ALL contain a supportedSettings directive.